collaborative-plonk/src/round1.rs

```diff
@@ -295,12 +295,13 @@ pub mod tests {
     use std::{fs::File, io::BufReader};
 
     use ark_bn254::Bn254;
-    use circom_types::{groth16::witness::Witness, plonk::ZKey, r1cs::R1CS};
-    use collaborative_groth16::{circuit::Circuit, groth16::SharedWitness};
+    use circom_types::plonk::ZKey;
+    use collaborative_groth16::groth16::SharedWitness;
```

Comment:
did we not want to put the sharedwitness into a separate crate? Other PR, right?

Reply:
Yes we did wanted to do that, but the plan was to do that in another PR. Also I didn't want to decide alone how we structure everything